cmd/system-probe/api/debug/handlers_linux.go

```diff
@@ -0,0 +1,29 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+//go:build linux
+
+// Package debug contains handlers for debug information global to all of system-probe
+package debug
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"os/exec"
+)
+
+// HandleSelinuxSestatus reports the output of sestatus as an http result
+func HandleSelinuxSestatus(w http.ResponseWriter, _ *http.Request) {
+	cmd := exec.Command("sestatus")
+	output, err := cmd.CombinedOutput()
+	// don't report ExitErrors since we are using the combined output which will already include stderr
+	if err != nil && !errors.Is(err, &exec.ExitError{}) {
+		fmt.Fprintf(w, "sestatus command failed: %s", err)
+		return
```

Comment:
Do you need to report a 500 status code for errors?

Reply:
I originally did 200 here since the FlareBuilder doesn't write the file in the event it 500s. But actually, it's probably fine to 500 since it goes to `flare_creation.log`. Updated